lib/tasks/import_prebakedstats.rake

```diff
@@ -0,0 +1,73 @@
+require 'csv'
+
+namespace :import do
+
+  desc "import CSV data into database"
+  task :prebakedstats, [:csv_file] => [:environment] do
+    habitats = Habitat.all
+    geo_entity_types = ["countries", "regions"].freeze
+
+    # import habitat data CSVs for each entity type
+    geo_entity_types.each do |geo_entity_type|
+      habitats.each do |habitat|
+        csv_file = "#{geo_entity_type}/#{habitat.name}.csv"
+        import_new_csv_file(geo_entity_type, habitat.name, csv_file)
+      end
+    end
+  end
+
+  def import_new_csv_file(geo_entity_type, habitat, csv_file)
+    filename = "#{Rails.root}/lib/data/#{csv_file}"
+    CSV.foreach(filename, headers: true, encoding: "utf-8") do |row|
+      if row.select {|k,v| /baseline/i =~ k}.any?
+        parse_change_stat row, geo_entity_type, habitat
+      else
+        parse_geo_entity_stat row, geo_entity_type, habitat
+      end
+    end
+  end
+
+  def parse_change_stat csv_row, geo_entity_type, habitat
+    geo_entity = fetch_geo_entity(csv_row["name"] | csv_row["Region"], csv_row["iso3"])
+    insert_change_stat(habitat, geo_entity, csv_row)
+  end
+
+  def parse_geo_entity_stat csv_row, geo_entity_type, habitat
+    geo_entity = fetch_geo_entity(csv_row["name"] | csv_row["Region"], csv_row["iso3"])
+    insert_geo_entity_stat(habitat, geo_entity, csv_row)
+  end
+
+  def insert_geo_entity_stat(habitat, geo_entity, csv_row)
+    habitat = Habitat.find_by(name: habitat)
+    ges = GeoEntityStat.create(habitat: habitat, geo_entity: geo_entity, protected_value: csv_row["total_value_protected"]&.strip || 0, 
```

Comment:
Two things here:

1. As the `ges` variable is not used anywhere, you can get rid of the assignment as the value on the right hand side will be returned anyway
2. I would probably put the computed `protected_value` into a variable first to clean up the code a bit.
Same below for `total_value` and `protected_percentage`.